### PR TITLE
First draft of values and behaviours

### PR DESCRIPTION
**Pull request by rcrphillips**
Not sure how this is going to work with a PR over a google doc, but please pile in. I think we should limit behaviours to 3 per value, so that we're not being overly prescriptive/descriptive.

### CODE REVIEW

**Comment by shtukas** [comment]
Nice!

**Comment by philmcmahon** [comment]
👍 thanks Rob

**Comment by AWare** [comment]
👍

File: values.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Knowledge Sharing
+ - Being forthcoming and broadcasting our work
+ - We strive to share what we work on with the wider department and communicate it’s value
```
Comment (paperboyo) [inline values.md]: > wider department
Just dept.? How about the whole business? Or even wider?

> it’s
its

File: values.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ - Being curious
+ 
+# Collaborative
+ - Working on problems together
+ - Pair programming
+ - Making sure decisions are made with everyone’s voices being heard
+ 
+# Inclusivity
+ - We conduct ourselves with humility, kindness and a calm manner
+ - Ideas can come from anyone on the team
```
Comment (paperboyo) [inline values.md]: > Ideas can come from anyone on the team

~on the team~?

**Pull request merged**